collections/slice.go

```diff
@@ -1,5 +1,9 @@
 package collections
 
+func All[T any](slice []T, predicate func(T) bool) bool {
+	return !Any(slice, predicate)
```

Comment:
Is this right? This would mean that the predicate is false for every item in the slice. 
```suggestion
	for _, val := range slice {
		if !predicate(val) {
			return false
		}
	}
	return true
```